### PR TITLE
Implement proper print handling and progress output

### PR DESCRIPTION
**Pull request by hgn**
@danielmgit:

non-verbose and verbose output variants to execute a campaign:

### CODE REVIEW

**Comment by hgn** [comment]
@danielmgit I will merge it when you ACK the commits, thank you

File: nas.py [hunk 1]
```diff
@@ -148,7 +148,7 @@ def run(self):
         # the status is used later for campaigns:
         # if the status is false the campaing must be
         # stopped, if true everything is fine!
-        print("  execute applet \"{} {}\"".format(self.applet_name, self.applet_args))
+        self.p.msg("  execute applet \"{} {}\"".format(self.applet_name, self.applet_args),                   level=Printer.STD)
```
Comment (danielmgit) [inline nas.py]: Something probably went wrong here?

File: nas.py [hunk 2]
```diff
@@ -267,6 +277,8 @@ def transform(self, content):
                 self.transform_exec_statement(chunks[1:], data)
             elif chunks[0] == "sleep":
                 self.transform_sleep_statement(chunks[1:], data)
+            elif chunks[0] == "print":
+                self.transform_print_statement(chunks[1:], data)
             else:
                 self.p.err("Command \"{}\" not known, only exec and sleep allowed".format(chunks[0]))
```
Comment (danielmgit) [inline nas.py:283]: This should also be updated at some point.

**Comment by danielmgit** [comment]
Looks nice, feel free to merge!

**Pull request merged**